examples/jupiter-swap/src/api/dscvr.ts

```diff
@@ -0,0 +1,59 @@
+import { Client, cacheExchange, fetchExchange } from '@urql/core';
+import { initGraphQLTada } from 'gql.tada';
+import { introspection } from '../graphql-env';
+
+export const graphql = initGraphQLTada<{
+  introspection: introspection;
+  scalars: {
+    DscvrId: string;
+    ContentId: string;
+  };
+}>();
+
+const getDataQuery = graphql(`
+  query getData($id: DscvrId!, $contentId: ContentId!, $hasContent: Boolean!) {
+    user(id: $id) {
+      id
+      username
+    }
+    content(id: $contentId) @include(if: $hasContent) {
+      id
+      creator {
+        username
+      }
+      contentType
+      portal {
+        name
+        iconUrl
+      }
+    }
+  }
+`);
+
+export const dscvrApiUrl = 'https://api1.stg.dscvr.one/graphql';
```

Comment:
should this be a constant or env variable ?

Reply:
Good point. @ncpenke and I discussed to have this as a as an ENV for the previous projects, we decided on having it as a const instead, so the user that downloads the repo can run it without setting env variables, like a standalone example.
BUT it seems I mixed up here because I'm setting `SOLANA_CHAIN_ID` and `JUPITER_API_URL` for my benefit also.

Having said that, here is my suggestion, we set it as an ENV variable with a default value, e.g:
```
export const dscvrApiUrl =  process.env.SOLANA_CHAIN_ID || 'https://api.dscvr.one/graphql';
```

Reply:
That sounds good should be `env.DSCVR_API_URL` right?

Reply:
Talked to Chandra, we will leave it hardcoded for now